src/cartridge_info.h

```diff
@@ -28,7 +28,7 @@
 /* Min and Max cartridge size in bytes */
 #define CARTRIDGE_MIN_SIZE        2048
 #define CARTRIDGE_MAX_SIZE	  (128 * 1024 * 1024)
-#define CARTRIDGE_LAST_SUPPORTED  70
+#define CARTRIDGE_LAST_SUPPORTED  74
```

Comment:
I wonder why this isn't part of the enum?

Reply:
I don't know... Could probably be refactored - should I do that?

Reply:
Sure, code improvements are always good. :)

Reply:
Done - as a separate commit so you can see my proposed changes.